### ISSUE TITLE
drupal.py need to look into default d8 path

### ISSUE THREAD

**Issue by mikkeschiren** [droope/droopescan#18]
In drupal.py we have:
```
    plugins_base_url = ["%ssites/all/modules/%s/",
            "%ssites/default/modules/%s/"]
    themes_base_url = ["%ssites/all/themes/%s/",
            "%ssites/default/themes/%s/"]
```

It should also look into modules folder for d8-sittes - that is default place to add modules in d8.

**Comment by droope** [comment]
Hey,

This sounds good. There are some docs for the new feature [here](https://www.drupal.org/docs/8/creating-custom-modules/naming-and-placing-your-drupal-8-module):

<pre>
Previous Drupal versions demanded custom modules to be located in /sites/all/modules as the core modules were located in /modules. However, in Drupal 8 /modules is now freed up for your custom and contributed modules. All core modules and libraries files are now located in the /core directory
</pre>

Ideally we'd only run this when version eight is detected but that might be a major change.

**Comment by droope** [comment]
I resolved this issue in droopescan 1.37.0. If you have the time to test it works I'd appreciate your testing.
